manim/animation/indication.py

```diff
@@ -104,7 +104,13 @@ def __init__(
         remover = True
         # Initialize with blank mobject, while create_target
         # and create_starting_mobject handle the meat
-        super().__init__(VMobject(), run_time=run_time, remover=remover, **kwargs)
+        if config.renderer == "opengl":
+            from manim.opengl import OpenGLVMobject
+
+            vmob_class = OpenGLVMobject
+        else:
+            vmob_class = VMobject
+        super().__init__(vmob_class(), run_time=run_time, remover=remover, **kwargs)
```

Comment:
I think you can just use `VGroup` instead.

Reply:
Good idea 👍